frontend/rolecall/src/app/performance/performance-editor.component.html

```diff
@@ -1,10 +1,10 @@
 <app-loading-spinner *ngIf="!dataLoaded"></app-loading-spinner>
 <div *ngIf="dataLoaded">
   <div *ngIf="performanceSelected; else selectPerformanceView">
-    <app-stepper #stepper
-        [ngStyle]="stepper.currentStepIndex == 3 && submitted ? { 'display': 'none' } : {}"
-        [stepperOptions]="stepperOpts"
-        (stepChange)="onStepChange($event)"></app-stepper>
+    <app-stepper #stepper [ngStyle]="stepper.currentStepIndex == 3 && submitted ? { 'display': 'none' } : {}" 
+      [stepperOptions]="stepperOpts" (stepChange)="onStepChange($event)"></app-stepper>
+
```

Comment:
Mikhail quoted studies that said it was easier to read or scan code that was narrower (you basically focus your eyes on one place and scroll). So we decided to do one attribute per line and write deep rather than wide code. This is a file where we haven't dome much (and we haven't done a cleanup pass). But we should not go backwards and reintroduce wide code.

Reply:
fixed